### PR TITLE
Adds the "Open" action

### PR DESCRIPTION
**Pull request by nkcr**
- updates GET `/elections/show/:electionId` to `/elections/:electionId`
- adds missing `Content-Type` and `CORS` header
- adds better error message for `useFetchCall`

The `Open` still won't work with the proxy because we need to perform the `DK1:Init` and `DK2:Setup` steps, which are not supported yet by the frontend.

### CODE REVIEW

File: web/frontend/src/components/utils/useChangeAction.tsx [hunk 1]
```diff
@@ -155,6 +170,17 @@ const useChangeAction = (
 
   const getAction = () => {
     switch (status) {
+      case STATUS.INITIAL:
+        return (
+          <span>
+            <button id="close-button" className="election-btn" onClick={handleOpen}>
+              {t('open')}
```
Comment (badrlarhdir) [inline web/frontend/src/components/utils/useChangeAction.tsx:177]: The `open` key is missing in the `en.json` file.

```ts
...
"open": "Open",
...

**Review by badrlarhdir — APPROVED** [review]
Excellent !

File: web/frontend/src/components/utils/useChangeAction.tsx (hunk 1 above)
Comment (nkcr) [reply web/frontend/src/components/utils/useChangeAction.tsx:177]: Nice catch!

**Review by cmsigrist — APPROVED** [review]
LGTM !

**Pull request merged**